modules/apps/frontend-js/frontend-js-state-web/build.gradle

```diff
@@ -5,4 +5,13 @@ dependencies {
 	compileOnly group: "org.osgi", name: "osgi.core", version: "6.0.0"
 	compileOnly project(":apps:frontend-js:frontend-js-loader-modules-extender-api")
 	compileOnly project(":core:petra:petra-string")
+}
+
+// TODO: figure out how/if this kind of thing can be applied globally
+// see: https://liferay.slack.com/archives/C04D8HMEF/p1616767736043200
+
+processResources {
+	exclude "**/*.d.js"
+	exclude "**/*.d.js.map"
+	exclude "**/*.ts"
```

Comment:
Note: just doing this in the one module while I wait for the answer about the "global" equivalent (or perhaps decide to move things into `types/`).